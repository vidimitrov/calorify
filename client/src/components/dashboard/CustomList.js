import styled from 'styled-components';

const MealsList = styled.div`
  padding: ${props => (props.top ? '32px' : '0 32px 32px')};
  flex-grow: 1;
  flex: 1;
  display: flex;
  flex-direction: column;
  height: 100%;
`;

export default MealsList;
