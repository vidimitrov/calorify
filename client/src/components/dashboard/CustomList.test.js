import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CustomList from './CustomList';

it('Dashboard/CustomList component renders correctly', () => {
  const tree = renderer.create(<CustomList />).toJSON();
  expect(tree).toMatchSnapshot();
});
