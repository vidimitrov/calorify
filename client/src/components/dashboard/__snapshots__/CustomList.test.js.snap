// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/CustomList component renders correctly 1`] = `
.c0 {
  padding: 0 32px 32px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100%;
}

<div
  className="c0"
/>
`;
